docs/cheatsheet/update.rst

```diff
@@ -0,0 +1,41 @@
+.. _ref_cheatsheet_update:
+
+Update
+======
+
+Flag all reviews to a specific movie:
+
+.. code-block:: edgeql
+
+    UPDATE Review
```

Comment:
Need an example of a `+=` operation; like the one we have on our front page

```
UPDATE
  Issue
FILTER
  # Get the Issue to be updated based
  # on the name.
  Issue.title = 'Update EdgeQL examples'

SET {
  time_spent_log := (
    # Add one more LogEntry to existing
    # logs.
    Issue.time_spent_log UNION (
      INSERT
        LogEntry {
          body := 'Fix typos',
          owner := Issue.owner,
          spent_time := 5
        }
    )
  )
};
```

but tailored for the movies schema

Reply:
hi @1st1,  is += operator implemented? Can i use with alpha2?

Reply:
No, not yet :( Hopefully in a3. In my comment I suggested that Victor should show how to emulate +=

Reply:
I've added a long-form example for incremental update.